ui/src/message_popup/playtest/smith_experience_page.jsx

```diff
@@ -28,13 +28,24 @@ export default React.createClass({
     query: React.PropTypes.shape({
       cohort: React.PropTypes.string,
       p: React.PropTypes.string
-    }).isRequired
+    }).isRequired,
+    facilitated: React.PropTypes.bool.isRequired
   },
 
   contextTypes: {
     auth: React.PropTypes.object.isRequired
   },
 
+
+
+
+  getDefaultProps() {
```

Comment:
You don't need this since you set this prop to `isRequired` up above in the `propTypes`.  This is only for if the prop is optional.